### PR TITLE
Add support for outcodes

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-139.dev.beta.nhschoices.net)

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "node-sass": "^4.5.3",
     "normalize.css": "^7.0.0",
     "nunjucks": "^3.0.1",
-    "postcodesio-client": "^0.1.2",
+    "postcodesio-client": "neilbmclaughlin/postcodesio-client-node#feature\/add-outcode-lookup",
```
Comment (beseven) [inline package.json]: I would usually ask for any forks to be in the org for visibility, but given you have asked for a PR in upstream, I think this is OK for now.

Comment (neilbmclaughlin) [inline package.json]: Yep, if I did this again I'd have the fork in nhsuk. I'm hoping it is short lived.

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-139.dev.beta.nhschoices.net)

**Pull request merged**